build.gradle

```diff
@@ -1,24 +1,47 @@
-// Top-level build file where you can add configuration options common to all sub-projects/modules.
-
 buildscript {
     repositories {
         maven { url "https://repository.exoplatform.org/public/" }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:2.0.0'
-
-        // NOTE: Do not place your application dependencies here; they belong
-        // in the individual module build.gradle files
+        classpath 'com.android.tools.build:gradle:3.0.1'
     }
 }
 
 allprojects {
+    buildscript {
+        repositories {
+            maven { url "https://repository.exoplatform.org/public/" }
+        }
+    }
     repositories {
         maven { url "https://repository.exoplatform.org/public/" }
         mavenLocal()
     }
+
+    afterEvaluate { project ->
+        // Load signing configuration
+        def propsFile = file(System.getenv('GOOGLE_PLAY_EXO_KEYSTORE_PROPS'))
```

Comment:
I have this error if I didn't set this variable (case of developers) :

> Error:(23, 1) A problem occurred configuring root project 'exo-android'.
path may not be null or empty string. path='null'

We should be able to launch the application in dev mode (emulator) without tweaking the build file IMO.